src/components/OrderDetails/OrderDetails.tsx

```diff
@@ -0,0 +1,47 @@
+import CheckIcon from '@mui/icons-material/Check';
+import CloseIcon from '@mui/icons-material/Close';
+import { Button } from '@mui/material';
+import ClickAwayListener from '@mui/material/ClickAwayListener';
+
+import styles from './OrderDetails.module.css';
+
+type OrederDetailsProps = {
+  handleClickAway: () => void;
+  dish: {
+    name: string;
+    quantity: number;
+    status: boolean;
+  }[];
+  note: string;
+};
+
+export function OrderDetails({ handleClickAway, dish, note }: OrederDetailsProps) {
+  const close = () => {
+    handleClickAway();
+  };
+
+  return (
+    <ClickAwayListener onClickAway={handleClickAway}>
+      <div className={styles.details}>
```

Comment:
The modal is wider than the width of a screen on phone sized screens. The app is intended for tablets so it shouldn't be a problem but if you have a quick solution for that, it'd encourage you to apply it - a client that might order such an app might change their mind in the future and you applied RWD for the rest of the elements there anyway.

Reply:
👍